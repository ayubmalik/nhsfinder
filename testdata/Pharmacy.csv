OrganisationID¬OrganisationCode¬OrganisationType¬SubType¬OrganisationStatus¬IsPimsManaged¬IsEPSEnabled¬OrganisationName¬Address1¬Address2¬Address3¬City¬County¬Postcode¬Latitude¬Longitude¬ParentODSCode¬ParentName¬Phone¬Email¬Website¬Fax
13043¬FA512¬Pharmacy¬Community Pharmacy¬Visible¬True¬True¬Lords Pharmacy ¬¬Unit 61, The Guineas Shopping Centre¬¬Newmarket¬Suffolk¬CB8 8EQ¬52.244796752929688¬0.4055977463722229¬Q56¬East Anglia Area Team¬01638 428022¬¬https://www.lordspharmacynewmarket.co.uk/¬01638 428022
13046¬FAP38¬Pharmacy¬Community Pharmacy¬Visible¬True¬True¬LloydsPharmacy Inside Sainsbury's¬Church Street¬Carlisle¬¬¬¬CA2 5TF¬54.894672393798828¬-2.9472866058349609¬Q49¬Cumbria, Northumberland, Tyne And Wear Area Team¬01228533915¬lp5045@lloydspharmacy.co.uk¬http://www.lloydspharmacy.com¬
13049¬FC826¬Pharmacy¬Community Pharmacy¬Visible¬True¬True¬Rutland Late Night Pharmacy ¬¬45c High Street¬¬Oakham¬¬LE15 6AJ¬52.670024871826172¬-0.7302858829498291¬Q59¬Leicestershire And Lincolnshire Area Team¬01572 723368¬info@rutland-pharmacy.com¬http://www.rutlandpharmacy.co.uk¬0872 1106511
13054¬FD294¬Pharmacy¬Community Pharmacy¬Visible¬True¬True¬Cohens Chemist¬33 Hill Street¬¬¬Hinckley¬Leicestershire¬LE10 1DS¬52.541034698486328¬-1.3677382469177246¬Q59¬Leicestershire And Lincolnshire Area Team¬01455613100¬branch.732@mycohens.net¬https://www.cohenschemist.co.uk/¬01455610512
