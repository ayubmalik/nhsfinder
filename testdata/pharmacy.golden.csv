FA512,Lords Pharmacy,,Unit 61 The Guineas Shopping Centre,,Newmarket,CB8 8EQ,01638 428022,,52.244796752929688,0.4055977463722229
FAP38,LloydsPharmacy Inside Sainsbury's,Church Street,Carlisle,,,CA2 5TF,01228533915,lp5045@lloydspharmacy.co.uk,54.894672393798828,-2.9472866058349609
FC826,Rutland Late Night Pharmacy,,45c High Street,,Oakham,LE15 6AJ,01572 723368,info@rutland-pharmacy.com,52.670024871826172,-0.7302858829498291
FD294,Cohens Chemist,33 Hill Street,,,Hinckley,LE10 1DS,01455613100,branch.732@mycohens.net,52.541034698486328,-1.3677382469177246